### ISSUE TITLE
How to change default middleware configs?

### ISSUE THREAD

**Issue by lucaswxp** [balderdashy/sails#2504]
I'm getting an error from multiparty library:

```Error: maxFields 1000 exceeded.```

But I truly need more than 1000 fields in my requests. How do I set this option inside sails?

**Comment by CWyrtzen** [comment]
Are you still getting this error?

**Comment by sailsbot** [comment]
Thanks for posting, @lucaswxp.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
